src/DssSpell.sol

```diff
@@ -18,185 +18,402 @@ pragma solidity 0.8.16;
+    address internal constant RWA014_A_OUTPUT_CONDUIT        = 0xD7cBDFdE553DE2063caAfBF230Be135e5DbB5064;
+
+    string  internal constant RWA014_DOC                     = "QmT2Dr1tTw4idtVXZHxjT5Cs22KsNJyZgmYy9LGf9kR7vU";
+    uint256 internal constant RWA014_A_INITIAL_PRICE         = 500 * MILLION * WAD;
+    uint48  internal constant RWA014_A_TAU                   = 0;
+    // Ilk registry params
+    uint256 internal constant RWA014_REG_CLASS_RWA           = 3;
+    // Remaining params
+    uint256 internal constant RWA014_A_LINE                  = 500 * MILLION;
+    uint256 internal constant RWA014_A_MAT                   = 100_00;
+    // Operator address
+    address internal constant RWA014_A_OPERATOR              = 0x3064D13712338Ee0E092b66Afb3B054F0b7779CB;
+    // Custody address
+    address internal constant RWA014_A_COINBASE_CUSTODY      = 0x2E5F1f08EBC01d6136c95a40e19D4c64C0be772c;
+    // -- RWA014 END --
+
+    // 24 May 2023 12:00:00 AM UTC
+    uint256 internal constant MAY_24_2023                    = 1684886400;
+    // 23 May 2023 11:59:59 PM UTC
+    uint256 internal constant MAY_23_2024                    = 1716508799;
+    // 23 May 2026 11:59:59 PM UTC
+    uint256 internal constant MAY_23_2026                    = 1779580799;
+
+    // Keeper Network
+    address internal constant GELATO_PAYMENT_ADAPTER         = 0x0B5a34D084b6A5ae4361de033d1e6255623b41eD;
+    address internal constant GELATO_TREASURY                = 0xbfDC6b9944B7EFdb1e2Bc9D55ae9424a2a55b206;
+    address internal constant KEEP3R_PAYMENT_ADAPTER         = 0xaeFed819b6657B3960A8515863abe0529Dfc444A;
+    address internal constant KEEP3R_TREASURY                = 0x4DfC6DA2089b0dfCF04788b341197146Ea97f743;
+    address internal constant CHAINLINK_PAYMENT_ADAPTER      = 0xfB5e1D841BDA584Af789bDFABe3c6419140EC065;
+    address internal constant TECHOPS_VEST_STREAMING         = 0x5A6007d17302238D63aB21407FF600a67765f982;
+
+    // Ecosystem Scope
+    address internal constant ECOSYSTEM_SCOPE_WALLET         = 0x6E51E0b5813152880C1389E3e860e69E06aD04D9;
+
+    // -- Spark GNO Onboarding components --
+    address internal constant SPARK_POOL_CONFIGURATOR        = 0x542DBa469bdE58FAeE189ffB60C6b49CE60E0738;
+    address internal constant SPARK_AAVE_ORACLE              = 0x8105f69D9C41644c6A0803fDA7D03Aa70996cFD9;
+    address internal constant SPARK_ATOKEN_IMPL              = 0x6175ddEc3B9b38c88157C10A01ed4A3fa8639cC6;
+    address internal constant SPARK_STABLE_DEBT_TOKEN_IMPL   = 0x026a5B6114431d8F3eF2fA0E1B2EDdDccA9c540E;
+    address internal constant SPARK_VARIABLE_DEBT_TOKEN_IMPL = 0x86C71796CcDB31c3997F8Ec5C2E3dB3e9e40b985;
+    address internal constant SPARK_INTEREST_RATE_STRATEGY   = 0x554265A713D6746A62d86A797254590784D436AA;
+    address internal constant SPARK_TREASURY                 = 0xb137E7d16564c81ae2b0C8ee6B55De81dd46ECe5;
+    address internal constant SPARK_GNO_ORACLE               = 0x4A7Ad931cb40b564A1C453545059131B126BC828;
```

Comment:
These addresses are not in the exec sheet / doc. should probably be added there by govalpha. What do you think about init-ing this process with them?

Reply:
This is the first week with SubDAO content inclusion - I am happy for us to push back on this if you are uncomfortable with it.

Reply:
Alright, thought about it for some time. Here we can skip this.